packages/cli/generation/ir-generator/src/converters/type-declarations/convertExampleType.ts

```diff
@@ -407,32 +407,28 @@ function convertObject({
                               throw new Error("Could not find original type declaration for property: " + wireKey);
                           }
 
-                          try {
-                              const valueExample = convertTypeReferenceExample({
-                                  example: propertyExample,
-                                  fileContainingExample,
-                                  rawTypeBeingExemplified:
-                                      typeof originalTypeDeclaration.rawPropertyType === "string"
-                                          ? originalTypeDeclaration.rawPropertyType
-                                          : originalTypeDeclaration.rawPropertyType.type,
-                                  fileContainingRawTypeReference: originalTypeDeclaration.file,
-                                  typeResolver,
-                                  exampleResolver
-                              });
-                              exampleProperties.push({
-                                  name: fileContainingExample.casingsGenerator.generateNameAndWireValue({
-                                      name: getPropertyName({
-                                          propertyKey: wireKey,
-                                          property: originalTypeDeclaration.rawPropertyType
-                                      }).name,
-                                      wireValue: wireKey
-                                  }),
-                                  value: valueExample,
-                                  originalTypeDeclaration: originalTypeDeclaration.typeName
-                              });
-                          } catch (e) {
-                              // skip properties that fail to convert (undiscriminated unions)
```

Comment:
we remove this catch all -- its possible stuff will throw on upgrade, but if it does folks can either downgrade or file an issue.